scripts/mac_intel_install.sh

```diff
@@ -56,25 +52,21 @@ echo "eval \"source \$(brew --prefix nvm)/nvm.sh\"" >> ~/.zshrc
 echo "eval \"\$(pyenv init --path)\"" >> ~/.zshrc
 echo "eval \"\$(pyenv init -)\"" >> ~/.zshrc
 echo "eval \"\$(direnv hook zsh)\"" >> ~/.zshrc
-echo "eval \"source \$(pyenv which virtualenvwrapper.sh)\"" >> ~/.zshrc
 echo "export WORKON_HOME=\"~/.virtualenvs\"" >> ~/.zshrc
-echo "export PATH=\"\$HOME/.poetry/bin:\$PATH\"" >> ~/.zshrc
+echo "export PATH=\"\$HOME/.local/bin:\$PATH\"" >> ~/.zshrc
 
 echo "${prefix}Reloading zsh"
 source ~/.zshrc
 
 
-echo "${prefix}Attemping to change pyenv version to 3.10.2"
-pyenv install 3.10.2
-pyenv global 3.10.2
 
 echo "${prefix}Attempting to change nvm version to v16.14.0(default)"
 nvm install v16.14.0
 nvm alias default v16.14.0
 nvm use default
 
 echo "${prefix}Attempting to install poetry"
-curl -sSL https://install.python-poetry.org | python3 -
+curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/install-poetry.py | python - --preview
```

Comment:
I think this needs to get updated now that poetry 1.2 is offical?

Reply:
Reverted now